### DISCUSSION TITLE
[Feature request] Customize profile page

### DISCUSSION THREAD

**Discussion by superm1991** [ChristopherKlay/StadiaEnhanced#103] [Ideas & Requests]
Hi Christopher,

I was looking at my profile page (specifically the game stats page https://stadia.google.com/profile/.../gameactivities/all ) and I think it would be useful to add some filters/ordering features.

Examples:
- sort by playtime
- sort by last played (current default?)
- sort by number of achievements
- sort by achievement percentage

**Comment by ChristopherKlay** [comment]
Looking into it and added to the project board (:
